src/lib.rs

```diff
@@ -84,4 +56,7 @@ pub mod subprotocols;
 #[doc(hidden)]
 pub mod circuit;
 pub mod misc;
```

Comment:
this is my bad: `misc` shouldn't have been pub form the start :/

Reply:
fixed